src/main/java/com/couchbase/client/dcp/conductor/HttpStreamingConfigProvider.java

```diff
@@ -67,7 +67,7 @@ public HttpStreamingConfigProvider(ClientEnvironment env) {
         super(LifecycleState.DISCONNECTED);
         this.env = env;
         this.remoteHosts = new AtomicReference<List<String>>(env.clusterAt());
-        this.configStream = PublishSubject.<CouchbaseBucketConfig>create().toSerialized();
+        this.configStream = BehaviorSubject.<CouchbaseBucketConfig>create().toSerialized();
```

Comment:
I don't think this can work, the publish subject has been chosen so that it waits until at least one config arises, the behavior subject will return null immediately. Does this just mask the effect that we are not getting a config in time??